pkgs/by-name/be/beekeeper-studio/package.nix

```
@@ -86,23 +95,42 @@ stdenv.mkDerivation (finalAttrs: {
     krb5
   ];
 
-  runtimeDependencies = map lib.getLib [ systemd ];
+  runtimeDependencies = lib.optionals stdenvNoCC.hostPlatform.isLinux (map lib.getLib [ systemd ]);
```

```suggestion
  runtimeDependencies = lib.optionals stdenvNoCC.hostPlatform.isLinux (lib.getLib systemd);
```
we don't need to run map on a one entry list